### ISSUE TITLE
Updated WHDLoad.hdf - adfDevType : unknown device type

### ISSUE THREAD

**Issue by tomjramsden** [libretro/libretro-uae#196]
Hello, when attempting to insert the kickstart roms into the recently released updated WHDLoad.hdf through ADF Opus I get a "adfDevType : unknown device type" therefore making it impossible to test the new release on the switch.

Thanks,
T

**Comment by sonninnos** [comment]
Did it work with the earlier WHDLoad.hdf..?

Here with Windows 10 that ancient ADF Opus is not working properly at all, and I've been transferring files successfully only with WinUAE and/or Total Commander with AmigaDX-plugin.

**Comment by MajorWilly** [comment]
I just tried transferring the Kickstarts to the new WHDLoad.hdf using ADF Opus and it worked fine.

**Comment by tomjramsden** [comment]
Okay, I installed total commander with AmigaDX plugin, and I get this error:
![image](https://user-images.githubusercontent.com/13732780/71640541-06d2b980-2c84-11ea-992d-bf25e2b2303e.png)

**Comment by tomjramsden** [comment]
> I just tried transferring the Kickstarts to the new WHDLoad.hdf using ADF Opus and it worked fine.

Hmmm... I am still getting this error?

![image](https://user-images.githubusercontent.com/13732780/71640619-5ebdf000-2c85-11ea-9491-2cccb1b3755a.png)

**Comment by tomjramsden** [comment]
Did it work with the earlier WHDLoad.hdf..?
 Here with Windows 10 that ancient ADF Opus is not working properly at all, and I've been transferring files successfully only with WinUAE and/or Total Commander with AmigaDX-plugin.

> But how about that good tip from MajorWilly: Use the WHDLoad.hdf once on some other working platform, so the Kickstarts will be transferred automatically, and then copy it to the Switch.

Unfortunately the workaround didn't work... :(

**Comment by sonninnos** [comment]
Looks like buildbot hasn't built the latest changes yet on Windows (both x64 and x64 cores are still dated 2019-12-30), so that needs to happen first for the new HDF to play ball.

So keep checking http://buildbot.libretro.com/nightly/ for the new build, then update the core, and then it should be gravy.

**Comment by tomjramsden** [comment]
> Looks like buildbot hasn't built the latest changes yet on Windows (both x86 and x64 cores are still dated 2019-12-30), so that needs to happen first for the latest HDF to play ball.
> 
> So keep checking http://buildbot.libretro.com/nightly/ for the new build, then update the core, and then it should be gravy.

Redownloaded the new whdload.hdf and all works now :) :+1:

**Comment by sonninnos** [comment]
Cool! I was just about to ask that surely you aren't using the HDF with the lines deleted in the Startup-Sequence, because of course that won't do anything with the Kickstarts then.

**Comment by tomjramsden** [comment]
All working now.
